internal/venice-common/src/main/java/com/linkedin/venice/pubsub/PubSubPositionUtils.java

```diff
@@ -0,0 +1,48 @@
+package com.linkedin.venice.pubsub;
+
+import com.linkedin.venice.pubsub.adapter.kafka.ApacheKafkaOffsetPosition;
+import com.linkedin.venice.pubsub.api.PubSubPosition;
+import java.nio.ByteBuffer;
+
+
+/**
+ * Utility class for converting PubSubPosition to and from byte arrays
+ */
+public class PubSubPositionUtils {
+  /**
+   * Convert a PubSubPosition to a byte array
+   * @param bytes the byte array representation of the position type and position value
+   * @return the position object
+   */
+  public static PubSubPosition fromBytes(byte[] bytes) {
+    ByteBuffer buffer = ByteBuffer.wrap(bytes);
+    int positionType = buffer.getInt();
+    if (positionType == PubSubPositionType.APACHE_KAFKA_OFFSET) {
+      return new ApacheKafkaOffsetPosition(buffer.getLong());
+    }
+    throw new IllegalArgumentException("Unknown position type: " + positionType);
+  }
```

Comment:
Let's have a proper Avro schema registered in `AvroProtocolDefinition` for this and leveraging `InternalAvroSpecificSerializer` for serde and versioning. Let's not cook up some hand-coded protocol please.

Reply:
Done.